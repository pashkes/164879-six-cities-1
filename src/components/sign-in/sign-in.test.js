import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import SignIn from './sign-in';
configure({adapter: new Adapter()});

it(`city link renders correctly`, () => {
  const currentCity = shallow(<SignIn />);
  expect(currentCity).toMatchSnapshot();
});
