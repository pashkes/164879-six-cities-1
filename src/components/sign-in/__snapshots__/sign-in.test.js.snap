// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`city link renders correctly 1`] = `
<WrapperForPages
  pageClasses={
    Array [
      "page",
      "page--gray",
      "page--login",
    ]
  }
>
  <ConnectFunction />
  <main
    className="page__main page__main--login"
  >
    <div
      className="page__login-container container"
    >
      <ConnectFunction />
      <section
        className="locations locations--login locations--current"
      >
        <div
          className="locations__item"
        >
          <a
            className="locations__item-link"
            href="#"
          >
            <span>
              Amsterdam
            </span>
          </a>
        </div>
      </section>
    </div>
  </main>
</WrapperForPages>
`;
